docs/index.html

```diff
@@ -1,58 +1,102 @@
-    </p>
-    <h3>Pull requests</h3>
-    <p>Test reports are sorted by last updated date.</p>
-    <!-- Create list of PR numbers -->
-    <ul>
-      {%- assign pr_pages = site.pages | where_exp: "page", "page.url contains
-      '/pr/'" | sort: "last_published" | reverse -%}
-      {%- assign pr_numbers = pr_pages | map: "pr_number" | uniq | compact -%}
-            
-      {%- for pr_num in pr_numbers -%}
-      <!-- Get PR title -->
-      {%- assign pr_pages_with_this_num = pr_pages | where: "pr_number", pr_num -%}
-      {%- assign pr_title_encoded = pr_pages_with_this_num[0].pr_title_encoded -%}
+<p class="lead">Test reports are sorted by last updated date.</p>
+<p class="alert alert-info">PR test runs are executed against the latest stable release of WordPress with the 2019 theme running on PHP 7.4 against localhost/wp-env.</p>
 
-      <!-- Create PR list item using the PR title and number-->
-      <li>
-        {{ pr_title_encoded | url_decode }}
-        <a href="https://github.com/woocommerce/woocommerce/pull/{{ pr_num }}"
-          >#{{ pr_num }}
-        </a>
-        <ul>
-          <!-- Get all the test types this PR has -->
-          {%- assign pr_test_types = pr_pages_with_this_num | map: "pr_test_type" | sort -%}
+<!-- Legend -->
+<details>
+  <summary><i>Legend</i></summary>
+  <table class="table table-dark table-striped" width="70%">
+    <thead>
+      <tr>
+        <td width="30%" style="text-align: center">
+          <strong>Symbol</strong>
+        </td>
+        <td width="70%"><strong>Meaning</strong></td>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td style="text-align: center">&#9989;</td>
+        <td>Test passed</td>
+      </tr>
+      <tr>
+        <td style="text-align: center">&#10060;</td>
+        <td>Test failed</td>
+      </tr>
+      <tr>
+        <td style="text-align: center">&#10068;</td>
+        <td>Test result unknown. Manually investigate.</td>
+      </tr>
+      <tr>
+        <td style="text-align: center">
+          <span style="color: gainsboro">API</span>
+        </td>
+        <td>(Grayed out text) Test not started, or in progress.</td>
+      </tr>
+    </tbody>
+  </table>
+</details>
 
-          <!-- For each test type, create a list item -->
-          {%- for test_type in pr_test_types -%}
-          <li>
-            <a href="pr/{{ pr_num }}/{{ test_type }}"
-              >{{ test_type | upcase }}</a
-            >
-          </li>
-          {%- endfor -%}
-        </ul>
-      </li>
+<br />
+
+<!-- Create list of PR numbers -->
+<table class="table table-dark table-striped">
+  {%- assign pr_pages = site.pages | where_exp: "page", "page.url contains
+  '/pr/'" | sort: "last_published" | reverse -%}
+  {%- assign pr_numbers = pr_pages | map: "pr_number" | uniq | compact -%}
+  <!-- Assign a temporary date value to prev_date -->
+  {%- assign prev_date = "Jan 1, 1969" | date: "%b %d, %Y" -%}
+        
```

Comment:
The logic in here gets kind of gnarly.  There's no built-in way to group things, so had to get creative with the "queries" to show PRs grouped by date.